default.nix

```diff
@@ -52,6 +52,8 @@ let
     inherit bspSrc l4tVersion;
   };
 
+  python-jetson = callPackage ./python-jetson.nix { };
```

Comment:
Could you switch this `callPackage` to `python3.callPackage`? Then your default.nix shouldn't need to accept `python3` as an argument, but rather `buildPythonPackage` and `pyftdi` directly.

Reply:
Good improvement, I just updated my PR